pkg/common/providers/ocmprovider/cluster.go

```diff
@@ -30,6 +37,16 @@ func (o *OCMProvider) LaunchCluster() (string, error) {
 	// we happen to forget to do it:
 	expiration := time.Now().Add(time.Duration(cfg.Cluster.ExpiryInMinutes) * time.Minute).UTC() // UTC() to workaround SDA-1567.
 
+	user, err := user.Current()
+
+	var username string
+	if err != nil {
+		log.Printf("Unable to get current user. Assuming this is being run from prow.")
+		username = "prow"
```

Comment:
I would add an additional check to verify this is run in Prow by checking the Job Name and Job ID (mainly Job ID) since those are only populated by Prow. That way all cases are covered.

Reply:
Sounds good. Just submitted an update with this in place.